specs/sale_spec.rb

```diff
@@ -0,0 +1,2 @@
+require_relative './spec_helper'
+require_relative '../lib/sale'
```

Comment:
Lets eliminate this line (and the similar ones in the other specs) in favor of requiring the module in the spec helper.